Project-I-Social-Card/social-card/src/App.js

```diff
@@ -1,22 +1,13 @@
 import React from 'react';
 import './App.css';
+import HeaderContainer from './components/HeaderComponents/HeaderContainer';
+import CardContainer from './components/CardComponents/CardContainer';
 
 const App = () => {
   return (
     <div>
-      <h3>Welcome to React Social Card!</h3>
```

Comment:
You could give this main div a width and margin 0 auto  to keep everything contained in the middle of the screen